package.json

```diff
@@ -37,6 +37,7 @@
   "dependencies": {
     "lodash": "^4.13.1",
     "moment": "^2.14.1",
+    "seamless-immutable": "^6.1.1",
```

Comment:
Works in Ember courtesy of my new shim [ember-seamless-immutable-shim](https://github.com/ciena-blueplanet/ember-seamless-immutable-shim)